src/error.rs

```diff
@@ -323,6 +323,14 @@ impl From<ParseError> for Error {
     }
 }
 
+impl From<&ParseError> for Error {
```

Comment:
Why is this From implemented on a ref instead of an owned error?

Reply:
It's implemented for both because `parsed_url` returns a ref. We can remove it if we decide to clone to an owned ParsedUrl.

Reply:
Gone